### PR TITLE
PPLUS-1478: We are not able to install spryker-sdk/sdk into the project with PHP 7.4 version

### PR DESCRIPTION
**Pull request by Kyrylo-Meretskyi**
Ticket: [PPLUS-1478](https://spryker.atlassian.net/browse/PPLUS-1478)

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -28,16 +28,17 @@
...
         "spryker-sdk/async-api": "^0.2.3",
         "spryker-sdk/composer-replace": "dev-master",
         "spryker-sdk/sdk-contracts": "^0.2.0",
         "spryker-sdk/spryk": "^0.4.0",
         "spryker-sdk/sync-api": "^0.1.1",
-        "spryker-sdk/upgrader": "dev-master",
-        "spryker/architecture-sniffer": "^0.5.2",
-        "spryker/code-sniffer": "^0.17.6",
+        "spryker-sdk/upgrader": "dev-bugfix/pplus-1478-sdk-installation-with-php74-project",
+        "spryker/architecture-sniffer": "dev-master",
```
Comment (vol4onok) [inline composer.json:40]: is it ok that we start to use dev versions?

**Review by vol4onok — DISMISSED** [review]

File: src/Extension/Task/DeprecationsCheckTask.yaml [hunk 2]
```diff
@@ -4,7 +4,7 @@ short_description: "Check your code by running a static code analysis and genera
 help: ~
 stage: build
 version: 1.0.0
-command: "php %sdk_dir%/vendor/bin/psalm --config=%config% --no-progress --output-format=json --report=%reportDir%/psalm.deprecations_report.json 1> /dev/null"
+command: "php %sdk_dir%/vendor/bin/psalm.phar --config=%config% --no-progress --output-format=json --report=%reportDir%/psalm.deprecations_report.json 1> /dev/null"
```
Comment (pavelmaksimov25) [inline src/Extension/Task/DeprecationsCheckTask.yaml:7]: why `*.phar` ? There is no `psalm.phar` in `vendor/bin`

Comment (Kyrylo-Meretskyi) [reply src/Extension/Task/DeprecationsCheckTask.yaml:7]: Because changed to another package to be compatible with spryker projects

Comment (pavelmaksimov25) [reply src/Extension/Task/DeprecationsCheckTask.yaml]: Is it tested with all existing Spryker projects: b2b, b2c shops, and suite-nonsplit?

**Comment by pavelmaksimov25** [comment]
Please merge the latest master branch changes and check the installation one more time.

**Comment by dereuromark** [comment]
whats the status on this? should either be merged or close now.

**Review by pavelmaksimov25 — APPROVED** [review]

**Pull request closed without merging**